train-tracker/src/index.js

```diff
@@ -3,15 +3,16 @@ import ReactDOM from 'react-dom/client';
 import './index.css';
 
 import App from './App';
-import Amtrak from './AmtrakAPI'
+import Amtrak, { Train } from './AmtrakAPI'
 import Map from './Map';
+import TrainList from './TrainList';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
     <App/>
     <Map/>
-
```

Comment:
In the future we'll want the Map component and the TrainList component within the App component, but obviously an easy fix and not a big deal rn